src/components/buttons.tsx

```diff
@@ -0,0 +1,42 @@
+import { useTheme } from "@mui/material";
+import Button from "@mui/material/Button";
+
+export const OutlinedButton = (props: any) => {
+  const theme = useTheme();
+  return (
+    <Button
+      variant="outlined"
+      sx={{
+        color: theme.palette.primary.main,
+        border: "1px solid #000000",
+        borderRadius: 100,
+        padding: "5px 14px",
+        fontSize: 14,
+        textTransform: 'none',
+        marginRight: 1,
+      }}
+    >
+      {props.children}
+    </Button>
+  );
+};
+
+export const ContainedButton = (props: any) => {
+  const theme = useTheme();
+  return (
+    <Button
+      variant="contained"
+      sx={{
+        color: theme.palette.primary.light,
+        backgroundColor: theme.palette.primary.dark,
```

Comment:
This was unnecessary because once you define the colors in the theme, you don't need to override them here.
If you comment out "color" & "backgroundColor", there would be no difference.
Also, separating a single component into two was't needed either as the "variant" prop handles it.

You can learn more about it here, https://mui.com/material-ui/customization/palette/#providing-the-colors-directly